.github/release-drafter-python-client.yml

```diff
@@ -0,0 +1,9 @@
+_extends: kaskada:.github/release-drafter.yml
```

Comment:
Some renaming to "free up" the name "python" for now.

I think in the next week or two it would be good if we stopped drafting releases for the python client and engine, and just had the single "kaskada release draft", but we can revisit once we get this out.